xbmc/utils/DatabaseUtils.cpp

```diff
@@ -741,6 +743,8 @@ int DatabaseUtils::GetField(Field field, const MediaType &mediaType, bool asInde
     else if (field == FieldGenre) index = VIDEODB_ID_TV_GENRE;
     else if (field == FieldMPAA) index = VIDEODB_ID_TV_MPAA;
     else if (field == FieldStudio) index = VIDEODB_ID_TV_STUDIOS;
+    else if (field == FieldTrailer)
+      index = VIDEODB_ID_TV_TRAILER;
```

Comment:
jenkins wanted this on newline

Reply:
This can be merged without Jenkins conformance for that line. But maybe `// clang-format off` surrounding the if-else block containing this line is better for future additions.

Reply:
is the latest change ok?

Reply:
Yeah, thanks.